price-feeder/oracle/convert.go

```diff
@@ -90,12 +90,19 @@ func convertCandlesToUSD(
 					return nil, err
 				}
 
+				// TODO: we should revise ComputeTVWAP to avoid return empty slices
+				// [#1261](https://github.com/umee-network/umee/issues/1261)
```

Comment:
```suggestion
				// TODO: we should revise ComputeTVWAP to avoid return empty slices
				// Ref: https://github.com/umee-network/umee/issues/1261
```